Tests/SwiftDocCTests/Infrastructure/PathHierarchyTests.swift

```diff
@@ -295,6 +294,11 @@ class PathHierarchyTests: XCTestCase {
             (symbolID: "s:14MixedFramework28CollisionsWithDifferentKindsO9somethingyA2CmF", disambiguation: "enum.case"),
             (symbolID: "s:14MixedFramework28CollisionsWithDifferentKindsO9somethingSSvp", disambiguation: "property"),
         ])
+        try assertPathRaisesErrorMessage("/MixedFramework/CollisionsWithDifferentKinds/something", in: tree, context: context, expectedErrorMessage: """
+        Reference is ambiguous after '/MixedFramework/CollisionsWithDifferentKinds': \
+        Add 'enum.case' to refer to 'case something'. \
```

Comment:
Two suggestions:
- Should we use 'Append' rather than 'Add' so that it's clear where to add the suffix?
- Should we include the hyphen, as in `'-enum.case'` so that it's clearer how to append the suffix?

Reply:
Yes, I think both of those would be good clarifications to the error message.

Reply:
Resolved in https://github.com/apple/swift-docc/pull/398/commits/0d3557fb4090dfcd8aed27d16604d72d78340374